tests/lava/proc/sparse/test_models.py

```diff
@@ -0,0 +1,588 @@
+# Copyright (C) 2021-22 Intel Corporation
```

Comment:
This entire file looks like a duplicate of the (public) unit test for Dense ProcModels. Is there a way to avoid this massive code duplication?

Reply:
It is. But why should it be avoided? The sparse process has to do the same thing that the dense process does? Therefore the same tests have to be conducted for this new process model